doc/nrf/libraries/nfc/tnep/poller.rst

```diff
@@ -38,12 +38,12 @@ The polling device can use single response communication mode to exchange the ND
 Exchanging data is possible only when a service is selected.
 The data is exchanged in the NDEF read procedure or the NDEF write procedure.
 
-To exchange data, use :c:func:`nfc_tnep_poller_svc_update` or :c:func:`nfc_tnep_poller_on_ndef_read`.
+To exchange data, use :c:func:`nfc_tnep_poller_svc_write` or :c:func:`nfc_tnep_poller_on_ndef_read`.
```

Comment:
@grochu and @KAGA164, I did not see a function called `nfc_tnp_poller_svc_update` in the header. I hope the `write` one is the correct function.